### ISSUE TITLE
Make invalid config error messages more informative

### ISSUE THREAD

**Issue by guilhermehn** [hudochenkov/postcss-sorting#65]
`at-variables` from [stylelint-order](https://github.com/hudochenkov/stylelint-order) is not considered a valid `order` value by [lib/validateOptions.js](https://github.com/hudochenkov/postcss-sorting/blob/e917e8f3db1cb89c51a27d0109008809b4863e80/lib/validateOptions.js#L158).

### postcss.js

```js
const fs = require('fs')
const postcss = require('postcss')
const sorting = require('postcss-sorting')

const config = {
	'order': [
		'at-variables'
	]
}

fs.readFile('a.css', (err, css) => {
	postcss([ sorting(config) ])
		.process(css, { from: 'a.css', to: 'b.css' })
		.then(result => {
			fs.writeFile('b.css', result.css)
		})
})
```

### a.css

```
body {
  color: #fff;
  font-size: 10px;
}
```

When I run `$ node postcss.js` the response is:

`postcss-sorting: Invalid "order" option value`

Is this intended behaviour?

**Comment by hudochenkov** [comment]
Yes, `at-variables` and `less-mixins` aren't supported in postcss-sorting. Even Less wasn't tested there. I'm not interested in working on Less support, but PRs are welcome.

**Comment by guilhermehn** [comment]
I do not need the Less support (who even uses Less these days?), but someone of my team just copy-pasted some predefined config when creating our shared stylelint-order config, which came with the `at-variables` value. We lost a good part of the day trying to figure out why the stylefmt was not working and just printing the error message.

It would be nice to at least have a friendlier error message, displaying invalid values that are present in the config.

**Comment by hudochenkov** [comment]
> who even uses Less these days?

I was surprised by that too. I know Wikipedia developers use it.

> It would be nice to at least have a friendlier error message, displaying invalid values that are present in the config.

Thank you for suggestion! I'll think how to make error messages more useful.

**Comment by hudochenkov** [comment]
Fixed in `3.0.2`. Thank you for a suggestion.

**Comment by guilhermehn** [comment]
Thank you!
